docs/admin/kubelet.md

```diff
@@ -111,6 +111,7 @@ kubelet
       --minimum-container-ttl-duration=1m0s: Minimum age for a finished container before it is garbage collected.  Examples: '300ms', '10s' or '2h45m'
       --network-plugin="": <Warning: Alpha feature> The name of the network plugin to be invoked for various events in kubelet/pod lifecycle
       --network-plugin-dir="/usr/libexec/kubernetes/kubelet-plugins/net/exec/": <Warning: Alpha feature> The full path of the directory in which to search for network plugins
+      --node-label=[]: add labels when registering the node in the cluster, the flag can be used multiple times (key=value or file:///full_path)
```

Comment:
This is a little too magic-y for my taste.  In other places we have the equivalent of
`--node-labels` and `--node-labels-file`, and they're mutually exclusive.